tests/store/api/find/test_find_business_data_sample.py

```diff
@@ -328,7 +328,7 @@ def test_get_number_of_reads_for_sample_from_metrics(
 
     # WHEN getting number of reads for a specific sample
     actual_total_reads = store_with_sequencing_metrics.get_number_of_reads_for_sample_from_metrics(
-        sample_internal_id=sample_id
+        sample_internal_id=sample_id, q30_threshold=0
```

Comment:
```suggestion
        sample_internal_id=sample_id, q30_threshold=0
```
Maybe have another threshold that does some filtering also instead of 0?

Reply:
Adding some tests for the filter 👍